server/pbenchinacan/run-pbench-in-a-can

```diff
@@ -67,6 +67,7 @@ sed -Ei \
     -e "/^ *realhost/ s/=.*/= $(hostname -f)/" \
     -e "s/<keycloak_realm>/${KEYCLOAK_REALM}/" \
     -e "s/<keycloak_client>/${KEYCLOAK_CLIENT}/" \
+    -e "s/##ADMIN_NAMES##/${PB_ADMIN_NAMES}/" \
```

Comment:
You'd rather use the `##...##` token delimiters than the `<...>` which the previous two expressions use?

Reply:
The idea was that an unaltered `admin_role = ##ADMIN_NAMES##` line is a *comment* with an empty value rather than an unlikely username.

Reply:
OK, that's kinda clever...although, it could potentially lead to a silent failure (i.e., it's _intended_ to remain quiet if unconfigured, which arguably is a bad thing) -- if the user wanted no admins, they could configure the substitution appropriately or remove the token from the file.  But...I'm not sure that using `<` instead of `##` produces a different result.